addon/components/markdown-to-html.js

```diff
@@ -25,7 +25,7 @@ const ShowdownComponent = Ember.Component.extend({
       }
     }
 
-    return new Handlebars.SafeString(this.converter.makeHtml(get(this, 'markdown')));
+    return Ember.String.htmlSafeg(this.converter.makeHtml(get(this, 'markdown')));
```

Comment:
@jasonmit what makes you think this only works in stable?  AFAIK this has existed for some time and this deprecation is the only new thing.